composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "shardj/zf1-future",
+    "name": "krytenuk/zf1-future",
```

Comment:
Mistake?

Reply:
Possibly, sorry I'm new to git and I needed to change this to include it in my project.

Reply:
Have reverted change

Reply:
Sorry new to git